stable/nginx-ingress/templates/controller-hpa.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if eq .Values.controller.kind "Deployment" }}
+{{- if .Values.controller.autoscaling }}
```

Comment:
Feels like this should be based on a boolean like `.Values.controller.autoscaling.enabled`

Reply:
changed!